cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2ClusterTest.java

```diff
@@ -322,8 +326,9 @@ public void testGenerateDeploymentWithTls() {
                     .withClusters(targetClusterWithTls)
                 .endSpec()
                 .build();
-        KafkaMirrorMaker2Cluster kmm2 = KafkaMirrorMaker2Cluster.fromCrd(resource, VERSIONS);
-        Deployment dep = kmm2.generateDeployment(emptyMap(), true, null, null);
+        KafkaMirrorMaker2Cluster kmm2 = KafkaMirrorMaker2Cluster.fromCrd(new Reconciliation("test", "kind", "namespace", "name"), resource, VERSIONS);
+        Deployment dep = kmm2.generateDeployment(
+                emptyMap(), true, null, null);
```

Comment:
Do we really need these split on two lines? It seems to be pretty short. Same in the occurrences of the same below.